File: old_trunk/cost/borland/Dynany/C++/Makefile.cpp
include ../../stdmk/stdmk

EXE = 	Client Server 

all: $(EXE)

clean:
	-rm -f core *.o *.hh *.cc $(EXE) 
	-rm -rf Templates.DB

#
# "Bank" specific make rules
#


Printer_c.cc: Printer.idl
	$(ORBCC) -type_code_info Printer.idl

Printer_s.cc: Printer.idl
	$(ORBCC) -type_code_info Printer.idl

Client: Printer_c.o Client.o 
	$(CC) -o Client Client.o \
	Printer_c.o $(LIBPATH) $(LIBORB) $(STDCC_LIBS)

Server: Printer_s.o Printer_c.o Server.o
	$(CC) -o Server Server.o \
	Printer_s.o Printer_c.o $(LIBPATH) $(LIBORB) $(STDCC_LIBS)

File: old_trunk/cost/borland/Dynany/Java/Makefile.java
# Makefile

.SUFFIXES: .java .class .idl .module

.java.class:
	vbjc $<

.idl.module:
	idl2java $<
	touch $@

default: all

clean: 
	rm -rf Printer 
	rm -f *.class *.tmp *.module *~

IDLS = \
	Printer.idl

MODULES = $(IDLS:.idl=.module) 

SRCS = \
	PrinterManagerImpl.java \
	Client.java \
	Server.java

CLASSES = $(SRCS:.java=.class) 

all:	$(MODULES) $(CLASSES)
